app/web/server.go

```diff
@@ -77,7 +77,7 @@ func sendErrorJSON(w http.ResponseWriter, r *http.Request, code int, err error,
 	render.JSON(w, r, JSON{"error": err.Error(), "details": details})
 }
 
-// GET /api/candle?from=2022-04-06T05:06:17.041Z&to=2022-04-06T06:06:17.041Z
+// GET /api/candle?from=2022-04-06T05:06:17.041Z&to=2022-04-06T06:06:17.041Z&max_points=100?files=10
```

Comment:
should be `&files=10`, max_points are not used.

Reply:
unless I'm missing something I do see [parsing](https://github.com/umputun/rlb-stats/blob/master/app/web/server.go#L109) of max_points and passing it (indirectly, but still) to [loadCandles](https://github.com/umputun/rlb-stats/blob/master/app/web/server.go#L118)